### PR TITLE
Update libraries

### PR DESCRIPTION
**Pull request by JesusMtnez**
- pureconfig 0.11.0
- http4s 0.20.1

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -18,9 +18,9 @@ lazy val V = new {
   val specs2: String           = "4.5.1"
   val enumeratum: String       = "1.5.13"
   val enumeratumCirce: String  = "1.5.21"
```
Comment (juanpedromoreno) [inline build.sbt]: Would you need to bump up enumeratumCirce?

File: build.sbt [hunk 2]
```diff
@@ -113,6 +113,7 @@ lazy val commonSettings = Seq(
   scalaVersion := "2.12.8",
   crossScalaVersions := Seq(scalaVersion.value),
   ThisBuild / scalacOptions -= "-Xplugin-require:macroparadise",
+  //parallelExecution in Test := false,
```
Comment (juanpedromoreno) [inline build.sbt]: Better to remove?

Comment (JesusMtnez) [reply build.sbt]: Ups, sorry :sweat:

File: build.sbt (hunk 1 above)
Comment (JesusMtnez) [reply build.sbt]: `enumeratum-circe` is already on the latest version

**Pull request merged**